gateway/controller/openresty/model/server.go

```diff
@@ -33,7 +33,14 @@ type Server struct {
 	Locations               []*Location
 	OptionValue             map[string]string
 	UpstreamName            string //used for tcp and udp server
-	ProxyStreamTimeout      string
+
+	// Sets the number of datagrams expected from the proxied server in response
+	// to the client request if the UDP protocol is used.
+	// http://nginx.org/en/docs/stream/ngx_stream_proxy_module.html#proxy_responses
+	// Default: 1
+	ProxyStreamResponses int
```

Comment:
Does this parameter take effect in template?

Reply:
Yes:

```
server {
	preread_by_lua_block {
		ngx.var.proxy_upstream_name="d67f5792189441fd9e99d18b8f469c2d_0_0_0_0_10000udp";
	}
	listen 0.0.0.0:10000 udp ;
	proxy_responses         1;
	proxy_timeout           600s;
	proxy_pass              upstream_balancer;
}
```